### PR TITLE
[TM ONLY] 24 Add Heaters to Air alarms

### PR DESCRIPTION
**Pull request by MissFox0810**
## About The Pull Request
Adds a really simple heating controller to the air alarm, that heats off a set point defined in the air alarms vars.

## Changelog
:cl: MissFox
add: Added built-in heater to air alarms
/:cl:

[Add Heaters to Air alarms](https://app.gitkraken.com/glo/card/7d520976f1134758aed2e23620324aaf)

### CODE REVIEW

**Review by Avunia — APPROVED** [review]

**Comment by hrzn-actions** [comment]
#### Test Merge Deployed

##### Server Instance
Horizon

##### Revision
Origin: a0a9e6b45172dcf1e96e4bca850d23602d726471
Pull Request: 041662689dc35f853fb674f7aeed172824d0f8b0
Server: 449939ba3e3d4d6984dd2ff1df5e6d0e88e3e0a1

**Comment by Azarak** [comment]
worried this would make the infinite delta active turfs flip flops even worse

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: a0a9e6b45172dcf1e96e4bca850d23602d726471
Pull Request: 63256f49706c058efbf18ff006280bd41c8bc001
Server: f6618b6ac1a67ce4b4147d4879df55bbb3099d2c

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 349b278d84cd3d8a322414997c472956743e5cb7
Pull Request: 63256f49706c058efbf18ff006280bd41c8bc001
Server: cbe06b66b49bb7c974d2218d88820060bc25be5c

File: tgui/packages/tgui/interfaces/AirAlarm.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ export const AirAlarm = (props, context) => {
         {!locked && (
           <AirAlarmControl />
         )}
+        {!locked && (
+          <AirAlarmHeatingControls />
```
Comment (Avunia) [inline tgui/packages/tgui/interfaces/AirAlarm.js:24]:
```suggestion
          <AirAlarmHeatingControls />
        )}
        {!locked && (
          <AirAlarmControl />
```

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 6659c04c1c6efd3a1471eb3e179435a1618fd3d8
Pull Request: 63256f49706c058efbf18ff006280bd41c8bc001
Server: e11684af276a99e1512acdfeae4e68394d394b40

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: 29dfda67469697557a2f95e66f356ebcdf0630fd
Pull Request: 51e4e8988900b43b3e80f174d1ace1e3e15085db
Server: 78af1b82f7ce609dffe2c75f5c49017e28890958

**Comment by hrzn-actions** [comment]
#### Test Merge Updated

##### Server Instance
Horizon

##### Revision
Origin: d787835eb02ad540ea3aa20f66064e6457ea9d0c
Pull Request: 51e4e8988900b43b3e80f174d1ace1e3e15085db
Server: e76b4ffa5f61205676e578a646fc5033ab630db8

**Comment by MissFox0810** [comment]
@Avunia done :3

**Review by Avunia — APPROVED** [review]

**Comment by hrzn-actions** [comment]
#### Test Merge Removed

**Pull request merged**